### PR TITLE
test(releases): testing release detail/summary screen

### PR DESCRIPTION
**Pull request by jordanl17**
### Description
* Adds tests for release details (the header)
* Adds tests for release summary (list of documents)
* Removes migration from name to slug script (as this has now run)

### CODE REVIEW

File: packages/sanity/src/core/releases/tool/Migrate.tsx
Comment (jordanl17) [inline packages/sanity/src/core/releases/tool/Migrate.tsx:1]: @pedrobonamin figured that we no longer needed this Migration script as we've migrated all the existing bundles to slug. Let me know if I'm wrong though!

File: packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+describe('ReleaseSummary', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+
+    await renderTest({})
+  })
+
+  it('lists the release title and description', () => {
+    screen.getByText('Release title')
+    screen.getByText('Release description')
```
Comment (RitaDias) [inline packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx:129]: just to double check, I think these tests need an expect statement right?
A getBy will work but it's not something we should likely rely on 🤔

**Review by bjoerge — APPROVED** [review]

File: packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx (hunk 1 above)
Comment (jordanl17) [reply packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx:129]: These tests are ok without an expect I believe. `getBy...` will throw if the element doesn't exist and fail the test. This is a pattern I remember being lauded many years ago when `testing-library/react` was first called `react-testing-library`. `.toBeInDocument` of course also works

Comment (RitaDias) [reply packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx:129]: huh 🤔 
Maybe it's a playwright thing (and not a jest / unit thing), but I know from their guidelines that they heavily suggest using expecting

**Pull request merged**